Mathlib/AlgebraicTopology/SimplexCategory/Defs.lean

```diff
@@ -167,6 +176,74 @@ noncomputable def inclusion.fullyFaithful (n : ℕ) :
 theorem Hom.ext {n} {a b : Truncated n} (f g : a ⟶ b) :
     f.toOrderHom = g.toOrderHom → f = g := SimplexCategory.Hom.ext _ _
 
+section Meta
```

Comment:
Could I encourage moving this to a separate file, perhaps even `Mathlib/Tactics/AlgebraicTopology/SimplexCategory.lean`?